lib/averages.js

```diff
@@ -22,4 +22,17 @@ function mean(values) {
   return average;
 };
 
-module.exports = {total, median, mean};
+function mode(values) {
+  var modes = {};
+  for(var i = 0; i < values.length; i++) {
+    var currentValue = values[i];
+    if (modes[currentValue] != undefined) {
+      modes[currentValue]++;
+    } else {
+      modes[currentValue] = 1;
+    };
+  };
+  return modes;
```

Comment:
after you tally how often a specific number occurs in the array, you'll need to find the highest frequency and return that specific number.

this is a solid start 👍